### ISSUE TITLE
Running superlinter locally never fails.

### ISSUE THREAD

**Issue by wrux** [github/super-linter#880]
I have followed [the setup steps](https://github.com/github/super-linter/blob/master/docs/run-linter-locally.md) for running the superlinter locally. 

If I run `docker run -e RUN_LOCAL=true -v $(pwd):/tmp/lint github/super-linter`, the superlinter works, but it never fails. I have created errors in my markdown, env and added a bunch of editorconfig errors; but nothing seems to cause the superlinter to fail. If I push my changes; the Github action works fine.

When running this command `docker run -e RUN_LOCAL=true -e LINTER_RULES_PATH=/tmp/lint -e VALIDATE_MARKDOWN=true -v $(pwd):/tmp/lint github/super-linter` here's the output:

```bash
2020-10-19 21:35:34 [INFO  ]   ---------------------------------------------
2020-10-19 21:35:34 [INFO  ]   --- GitHub Actions Multi Language Linter ----
2020-10-19 21:35:34 [INFO  ]    - Image Creation Date:[2020-10-19T15:28:55Z]
2020-10-19 21:35:34 [INFO  ]    - Image Revision:[92a2391]
2020-10-19 21:35:34 [INFO  ]    - Image Version:[latest]
2020-10-19 21:35:34 [INFO  ]   ---------------------------------------------
2020-10-19 21:35:34 [INFO  ]   ---------------------------------------------
2020-10-19 21:35:34 [INFO  ]   The Super-Linter source code can be found at:
2020-10-19 21:35:34 [INFO  ]    - https://github.com/github/super-linter
2020-10-19 21:35:34 [INFO  ]   ---------------------------------------------
2020-10-19 21:35:34 [INFO  ]   --------------------------------------------
2020-10-19 21:35:34 [INFO  ]   Gathering GitHub information...
2020-10-19 21:35:34 [INFO  ]   NOTE: ENV VAR [RUN_LOCAL] has been set to:[true]
2020-10-19 21:35:34 [INFO  ]   bypassing GitHub Actions variables...
2020-10-19 21:35:34 [INFO  ]   Linting all files in mapped directory:[/tmp/lint]
2020-10-19 21:35:34 [INFO  ]   Successfully found:[GITHUB_TOKEN]
2020-10-19 21:35:34 [INFO  ]   --------------------------------------------
2020-10-19 21:35:34 [INFO  ]   Gathering user validation information...
2020-10-19 21:35:39 [INFO  ]   ---------------------------------
2020-10-19 21:35:39 [INFO  ]   ------ File list to check: ------
2020-10-19 21:35:39 [INFO  ]   ---------------------------------
2020-10-19 21:35:39 [INFO  ]   ----------------------------------------------
2020-10-19 21:35:39 [INFO  ]   Successfully gathered list of files...
2020-10-19 21:35:39 [INFO  ]   
2020-10-19 21:35:39 [INFO  ]   ----------------------------------------------
2020-10-19 21:35:39 [INFO  ]   ----------------------------------------------
2020-10-19 21:35:39 [INFO  ]   Linting [MARKDOWN] files...
2020-10-19 21:35:39 [INFO  ]   ----------------------------------------------
2020-10-19 21:35:39 [INFO  ]   ----------------------------------------------
2020-10-19 21:35:39 [INFO  ]   ----------------------------------------------
2020-10-19 21:35:39 [INFO  ]   ----------------------------------------------
2020-10-19 21:35:39 [INFO  ]   Generated reports:
2020-10-19 21:35:39 [INFO  ]   ----------------------------------------------
2020-10-19 21:35:39 [INFO  ]   ----------------------------------------------
2020-10-19 21:35:39 [INFO  ]   Reports generated in folder /tmp/lint/super-linter.report
2020-10-19 21:35:39 [WARN  ]   Report output folder (/tmp/lint/super-linter.report) does NOT exist.
2020-10-19 21:35:39 [INFO  ]   ----------------------------------------------
2020-10-19 21:35:39 [INFO  ]   ----------------------------------------------
2020-10-19 21:35:39 [INFO  ]   The script has completed
2020-10-19 21:35:39 [INFO  ]   ----------------------------------------------
2020-10-19 21:35:39 [INFO  ]   ----------------------------------------------
2020-10-19 21:35:39 [NOTICE]   All file(s) linted successfully with no errors detected
2020-10-19 21:35:39 [INFO  ]   ----------------------------------------------
```

**Comment by ferrarimarco** [comment]
I *think* this is expected, because when running locally, we don't load configuration from the GitHub actions file.

**Comment by GaboFDC** [comment]
I think the title is misleading.
In local it is expected to don't detect GitHub actions stuff, as is not running as a GitHub action.
But in local this should fail if the linter detects something. I'm giving it a look. but I think the title should be something like "running super-liner locally, doesn't fail"

**Comment by GaboFDC** [comment]
Seems related to #867 fixed in #884

**Comment by wrux** [comment]
@GaboFDC I've changed the title and description to something more relevant.
